src/CoreArbiterServer.cc

```diff
@@ -805,7 +805,10 @@ CoreArbiterServer::timeoutThreadPreemption(int timerFd)
 void
 CoreArbiterServer::cleanupConnection(int socket)
```

Comment:
I wonder if the argument to this method should be a ProcessInfo rather than an int. Let's discuss.